tools/marvin/marvin/config/test_data.py

```diff
@@ -781,6 +781,15 @@
         "ostype": "CentOS 5.6 (64-bit)"
 
     },
+    "coreos_volume": {
+        "diskname": "Volume_core",
+        "urlvmware":"http://10.147.28.7/templates/coreos/coreos_production_vmware.ova",
```

Comment:
@remibergsma  Removed the link and added a public link  like 
http://dl.openvm.eu/cloudstack/coreos/x86_64/coreos_production_cloudstack_image-vmware.ova